hack/build/docker/builder/Dockerfile

```diff
@@ -1,14 +1,20 @@
-FROM quay.io/centos/centos:stream9
+FROM registry.fedoraproject.org/fedora-minimal:33
 MAINTAINER "The KubeVirt Project" <kubevirt-dev@googlegroups.com>
 
-RUN 	dnf -y install dnf-plugins-core && \
-	dnf config-manager --set-enable crb && dnf update -y && \
-	dnf install -y \
+COPY fedora.repo /tmp/fedora_ci.dnf.repo
```

Comment:
This was for oVirt CI as well, don't need it, as well as the repo file is not needed.